core/src/main/java/greencity/config/SecurityConfig.java

```diff
@@ -104,7 +104,7 @@ protected void configure(HttpSecurity http) throws Exception {
                 "/ownSecurity/signUp",
                 "/ownSecurity/signIn",
                 "/ownSecurity/updatePassword",
-                "/email/addEcoNews",
+//                "/email/addEcoNews",
```

Comment:
I think, /email/addEcoNews need to add between antMatchers and hasAnyRole because in all authorization levels (User, Admin, etc.)response back 403

Reply:
Also you need to delete the comment